WordPress/src/main/java/org/wordpress/android/ui/reader/ReaderPostRenderer.java

```diff
@@ -307,6 +310,9 @@ private void replaceIframeTag(final String tag, final String src) {
      * returns the full content, including CSS, that will be shown in the WebView for this post
      */
     private String formatPostContentForWebView(final String content, boolean renderAsTiledGallery) {
```

Comment:
The changes below look good, but I'm wondering if we can also change the `renderAsTiledGallery` parameter so it's only true when the post actually contains a gallery.